pkg/query/internal/test-storeset-pre-v0.8.0/storeset.go

```diff
@@ -207,6 +208,18 @@ func (s *storeRef) TimeRange() (int64, int64) {
 	return s.minTime, s.maxTime
 }
 
+func (s *storeRef) LabelNamesBloom() bloom.Filter {
```

Comment:
This is a very old version of the store, let's just return an `AlwaysTrue` filter.